Yoast/wordpress-seo
# Fixed wrong object propery reference

## Summary

This PR can be summarized in the following changelog entry:

*[not-userfacing] Fixes php errors when updating plugins.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Update plugins, make sure there are no errors.
```
Notice: Undefined property: stdClass::$storeUrl in /srv/www/wordpress-default/public_html/wp-content/plugins/wordpress-seo-premium/inc/class-addon-manager.php on line 248

Notice: Undefined property: stdClass::$lastUpdated in /srv/www/wordpress-default/public_html/wp-content/plugins/wordpress-seo-premium/inc/class-addon-manager.php on line 249

Notice: Undefined property: stdClass::$storeUrl in /srv/www/wordpress-default/public_html/wp-content/plugins/wordpress-seo-premium/inc/class-addon-manager.php on line 250
```

## UI changes

## Quality assurance


Fixes #
